parts/linux/cloud-init/artifacts/ci-syslog-watcher.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+
+[ ! -f "/var/run/mdsd/update.status" ] && exit 0
+status=$(cat /var/run/mdsd/update.status)
+
+if [[ "$status" == "add" ]]; then
+        echo "Status changed to $status."
+        cat >/etc/rsyslog.d/60-rsyslog-forward-mdsd.conf << 'EOL'
+$ModLoad omuxsock 
+$OMUxSockSocket /var/run/mdsd/default_syslog.socket 
+template(name="MDSD_RSYSLOG_TraditionalForwardFormat" type="string" string="<%PRI%>%TIMESTAMP% %HOSTNAME% %syslogtag%%msg:::sp-if-no-1st-sp%%msg%")
+$OMUxSockDefaultTemplate MDSD_RSYSLOG_TraditionalForwardFormat
+*.* :omuxsock:
+EOL
+elif [[ "$status" == "remove" ]]; then
+        echo "Status changed to $status."
+        [ -f "/etc/rsyslog.d/60-rsyslog-forward-mdsd.conf" ] && rm /etc/rsyslog.d/60-rsyslog-forward-mdsd.conf
```

Comment:
dumb question, have you tried this with mariner // does Azure Monitor support mariner?

you've added it to all the mariner vhds as well, please either confirm it works there, or remove it from those files. As we open up mariner to 3P we should probably doc the limitation if it exists.

Reply:
We will support mariner. I tested it with mariner v1 and v2